pkg/modelzooserver/modelzooserver.go

```diff
@@ -15,28 +15,162 @@ package modelzooserver
 
 import (
 	"context"
+	"fmt"
+	"io"
+	"log"
 
+	"sqlflow.org/sqlflow/pkg/database"
 	pb "sqlflow.org/sqlflow/pkg/proto"
 )
 
+const modelCollTable = "sqlflow_model_zoo.model_collection"
```

Comment:
How about col, and collection => repo?

Reply:
Maybe the name "repo" would let model contributors be confused about the model source code stored in git repo?